afk-daily.sh

```diff
@@ -1945,12 +1962,15 @@ run() {
     if checkToDo doStrengthenCrystal; then strengthenCrystal; fi
     if checkToDo doTempleOfAscension; then templeOfAscension; fi
     if checkToDo doCompanionPointsSummon; then nobleTavern; fi
-    if checkToDo doCollectOakPresents; then oakInnSpeedy; fi
+    if checkToDo doCollectOakPresents; then oakInn; fi
 
     # END
     if checkToDo doCollectQuestChests; then collectQuestChests; fi
     if checkToDo doCollectMail; then collectMail; fi
-    if checkToDo doCollectMerchantFreebies; then collectMerchants; fi
+    if checkToDo doCollectMerchantFreebies; then
+        collectMerchants
+        doCollectMerchantFreebies=false  # Prevent loop on error
```

Comment:
On error, it was looping because this was never set to false :/

Other are set to false when checkToDo is called with another parameter, but since it's the last one, it was buggy :/
Not a clean solution, need to work on it ^^